acceptance-tests/src/test/scala/ch/renku/acceptancetests/HandsOnSpec.scala

```diff
@@ -20,64 +20,47 @@ package ch.renku.acceptancetests
 
 import ch.renku.acceptancetests.model.projects.ProjectDetails
 import ch.renku.acceptancetests.pages._
-import ch.renku.acceptancetests.tooling.{AcceptanceSpec, DocsScreenshots, KnowledgeGraphApi}
+import ch.renku.acceptancetests.tooling.{AcceptanceSpec, KnowledgeGraphApi}
 import ch.renku.acceptancetests.workflows._
-import org.openqa.selenium.JavascriptExecutor
 
 import scala.concurrent.duration._
-import scala.language.postfixOps
 
 /**
   * Run the HandsOn from the documentation.
   */
 class HandsOnSpec
     extends AcceptanceSpec
-    with Collaboration
     with Environments
     with Login
-    with NewProject
-    with RemoveProject
+    with Project
```

Comment:
Are there specs which don't require this trait?

Reply:
Yes, like LoginSpec